pegasus/sites.v3/code.org/public/project-ideas.md.partial

```diff
@@ -0,0 +1,370 @@
+            Project Guide
+          </a>
+        </p>
+      </td>
+      <td>
+        <h3><strong>Journal or Blog</strong></h3>
+        <p>Create a space for you to write about your experiences. Add a new page each daily or week!</p>
+        <img src="/images/athome/journal.png" style="max-width:50%"/>
+        <br>
+        <br>
+        <p>
+          <a href="https://docs.google.com/document/d/104TYIDCUq8n7AiSEYFMqdkBqes8Yr9WCi1zqEdoBSkU/edit#">
+            More Details
+          </a>
+        </p>
+        <p>
+          <a href="https://docs.google.com/document/d/1bMPvQTEtThFx5R0yIoJN4290SerK9lLgtPXp5zS-umQ/edit#">
+            Project Guide
+          </a>
+        </p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+<div style="clear: both; margin-bottom: 40px"></div>
+
+## [**Game Lab**](https://code.org/educate/gamelab)
+<div class="col-33" style="padding-right: 20px;">
+  <img src="/images/athome/game-lab.jpg">
+</div>
+<div class="col-66">
+  <p>
+    Game Lab is a programming environment where you can make simple animations and games with objects and characters that interact with each other. Recommended for students who have completed at least Chapter 1 of Interactive Animations and Games from <a href="https://studio.code.org/courses/csd">CS Discoveries</a> or the <a href="https://studio.code.org/s/csd3-virtual">Self-Paced Introduction to Game Lab</a>.
+  <p>
+  <br>
+  <p>
+    <a href="https://studio.code.org/docs/gamelab/">
+      Game Lab Documentation
+    </a>
+  </p>
+</div>
+<div style="clear: both; margin-bottom: 20px"></div>
+<table>
+  <thead>
+    <tr>
+      <th>Game Lab Project Ideas</th>
+      <th></th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <h3><strong>No Sprites Animation Challenge</strong></h3>
+        <p>
+          Test your understanding of variables and the draw loop by
+          creating an animated scene without any sprites!
+        </p>
+        <img src="/images/athome/no-sprites.gif" style="max-width:50%"/>
+        <br>
+        <br>
+        <p>
+          <a href="https://docs.google.com/document/d/1JGmqwOUr-9LgjbbaEqVK8AQsHSQWvyjgHOJ-9CblU6o/edit#heading=h.jornae4vwl7p">
+            More Details
+          </a>
+        </p>
+        <p>
+          <a href="https://docs.google.com/document/d/1PyqSvPvg01usv6uS1lf-E_kLHAf59iWteyALUqOlEgM/edit#">
+            Project Guide
+          </a>
+        </p>
+      </td>
+      <td>
+        <h3>
+          <strong>Randomized Image Creator</strong>
+        </h3>
+        <p>
+          Apply knowledge of user input and the draw loop to create on-
+          demand randomized images.
+        </p>
+        <img src="/images/athome/random-image.gif" style="max-width:50%"/>
+        <br>
+        <br>
+        <p>
+          <a href="https://docs.google.com/document/d/
+            1QGIvapBacMyvHVRDrLH179DEkjOMUBrv7Zh6OYz_Vyo/edit#">
```

Comment:
Also something I noticed on staging - it looks like this link doesn't work? In general, I think something weird with formatting happened - any of the links with blank lines on GitHub aren't working on staging?
![image](https://user-images.githubusercontent.com/68714964/140987026-dcac7ba4-1437-4771-953d-b5fc888e9373.png)